sdk/containerregistry/Azure.Containers.ContainerRegistry/perf/Azure.Containers.ContainerRegistry.Perf/Infrastructure/PerfTestEnvironment.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core.TestFramework;
+
+namespace Azure.Containers.ContainerRegistry.Perf
+{
+    /// <summary>
+    /// Represents the ambient environment in which the test suite is being run, offering access to information such as environment variables.
+    /// </summary>
+    internal sealed class PerfTestEnvironment : TestEnvironment
```

Comment:
Can the ContainerRegistryEnvironment be used instead?

Reply:
It uses recorded variables?  Honestly, I was following the README and Search and FormRecognizer examples.  Beyond a small amount of code duplication, is it a concern?

Reply:
No, I don't think it is a concern -  I was mostly curious why there was a separate TestEnvironment. I see that you don't actually have a dependency on your test project from here so it may not make sense to do so. 

FWIW, I don't think it would matter that the environment uses Recorded variables as the `RecordedTestMode? Mode` property would end up being null (I think).